eclass/node.eclass

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EXPORT_FUNCTIONS src_prepare src_compile src_install
+
+RESTRICT+=" mirror"
+SLOT="0"
+
+[ -z "${SRC_URI}" ] && SRC_URI="https://registry.npmjs.org/${PN}/-/${P}.tgz"
+
+[ -z "${NODEJS_BDEPEND}" ] && BDEPEND+=" ${NODEJS_BDEPEND}"
+[ -z "${NODEJS_RDEPEND}" ] && RDEPEND+=" ${NODEJS_RDEPEND}"
+[ -z "${NODEJS_DEPEND}" ] && DEPEND+=" ${NODEJS_DEPEND}"
+[ -z "${RDEPEND}" ] && DEPEND+=" ${RDEPEND}"
```

Comment:
This is probably not necessary, if the variable is set in the ebuild it overwrites the variable set in the eclass anyway, so I think you can just set it unconditionally in the eclass.

Reply:
I'm not getting it. What do you propose to write?

Reply:
> I'm not getting it. What do you propose to write?

I meant the same thing that @telans suggested below. There's no need to check if the variable is set, ebuild variables overwrite eclass variables anyway, so there is no risk of the eclass overwriting anything.

Reply:
This code is no more

Reply:
This code is no more